runtime/configs/config.go

```diff
@@ -120,6 +120,7 @@ func NewDefaultConfig(options ...func(*Config)) *Config {
 			},
 		},
 		Utility: &UtilityConfig{
+			// TODO: add a servicer default config
```

Comment:
Should we have an empty struct here for now? You can add an `enabled` flag in the config and set it to `false`

Reply:
Thanks, Done.